examples/wagmi/src/index.tsx

```diff
@@ -18,7 +18,7 @@ if (!window.Buffer) {
   window.Buffer = Buffer;
 }
 
-const alchemyId = process.env.REACT_APP_ALCHEMY_ID;
+const alchemyId = process.env.REACT_APP_ALCHEMY_ID || 'UuUIg4H93f-Bz5qs91SuBrro7TW3UShO';
```

Comment:
should we hardcode this alchemy key? (just to make sure that its a dev key)

Reply:
Copilot got me this one, I don't own it :D it works. Wagmi require a key in 0.7